server/src/resolvers/models.ts

```diff
@@ -5,6 +5,22 @@ export const models = [
       publishCreate: true,
       publishUpdate: true,
       publishDelete: true
+    },
+    // this auth config piece should be generated by the graphback plugin
+    // we can discuss whether that's directives or annotations
+    auth: {
```

Comment:
It will be annotations. The only deal here is that his will be convoluted in terms of packages:

- Generator package will be required in devDependencies
- Service package will be required in dependencies to implement KeycloakService

Reply:
Challenge to compose this: Resolves plugin will generate models but then separate plugin will need to overwrite it in some specific way.

Reply:
Yeah. I think that something similar should be done with how the schema is handled, it would be cool if you could pass the models through the plugins before they get written